client/src/components/Ticketing/ticketingmanager/Season/components/SeasonEvents.tsx

```diff
@@ -1,7 +1,229 @@
+      eventname: eventName,
+      eventdescription: eventDescription,
+      seasonid_fk: null,
+    };
+    delete eventToUpdate['id'];
+    delete eventToUpdate['seasonid'];
+    delete eventToUpdate['seasonticketeligible'];
+    delete eventToUpdate['numshows'];
+    delete eventToUpdate['description'];
+    delete eventToUpdate['title'];
+
+    const updateEventCall = await updateEventSeason(eventToUpdate, token);
+    if (updateEventCall) {
+      setAllEventInfo(updatedEvents);
+      setEventsNotInSeason([...eventsNotInSeason, eventToUpdateCopy]);
+    }
+  };
+
+  const deleteEventConfirmationHandler = (eventId: number) => {
+    setShowPopUp(true);
+    setPopUpMessage({
+      title: 'Remove event',
+      message: 'Are you sure you want to remove this event?',
+      success: false,
+      handleClose: () => setShowPopUp(false),
+      handleProceed: () => {
+        handleRemoveEvent(eventId);
+        setShowPopUp(false);
+      },
+    });
+  };
+
+  const handleAddEventToSeason = async (eventIdToAdd: number) => {
+    let eventToAdd = eventsNotInSeason.find(
+      (event) => Number(event.id) === eventIdToAdd,
+    );
+    const eventToAddCopy = {...eventToAdd};
+    const updatedEventsNotInSeason = eventsNotInSeason.filter((event) => {
+      return Number(event.id) !== eventIdToAdd;
+    });
+
+    // Temporary solution until Events API is updated (Line 94 - 113)
+    // Fetch all event API returns data with different property names than what update event API requires as payload
+    const {
+      id: eventId,
+      description: eventDescription,
+      title: eventName,
+    } = eventToAdd;
+    eventToAdd = {
+      ...eventToAdd,
+      eventid: eventId,
+      eventname: eventName,
+      eventdescription: eventDescription,
+      seasonid_fk: seasonId,
+    };
+    delete eventToAdd['id'];
+    delete eventToAdd['seasonid'];
+    delete eventToAdd['seasonticketeligible'];
+    delete eventToAdd['numshows'];
+    delete eventToAdd['description'];
+    delete eventToAdd['title'];
+
+    const updateEventCall = await updateEventSeason(eventToAdd, token);
+    setShowPopUp(true);
+    if (updateEventCall) {
+      setAllEventInfo([...allEventInfo, eventToAddCopy]);
+      setEventsNotInSeason(updatedEventsNotInSeason);
+      setPopUpMessage({
+        title: 'Success',
+        message: 'The event has been added to the season!',
+        success: true,
+        handleClose: () => setShowPopUp(false),
+        handleProceed: () => setShowPopUp(false),
+      });
+    } else {
+      setPopUpMessage({
+        title: 'Error',
+        message:
+          'There was a error adding the event to the season. Pleas try again.',
+        success: false,
+        handleClose: () => setShowPopUp(false),
+        handleProceed: () => setShowPopUp(false),
+      });
+    }
+  };
+
+  useEffect(() => {
+    void handleGetAllEvents();
+  }, []);
+
+  return (
+    <div className='rounded-xl p-7 bg-white text-lg mt-5'>
```

Comment:
Could you give the `shadow-xl` class to this div, so it appears similar to the events page

Reply:
Done!